set.mm

```diff
@@ -80882,354 +81079,532 @@ C Fn ( ( S i^i dom F ) u. { z } ) ) $=
   $}
 
   ${
     $d R x $.  $d A x $.  $d X x $.  $d G x $.  $d F x $.
-    wfr2a.1 $e |- R We A $.
-    wfr2a.2 $e |- R Se A $.
-    wfr2a.3 $e |- F = wrecs ( R , A , G ) $.
+    wfr2aOLD.1 $e |- R We A $.
+    wfr2aOLD.2 $e |- R Se A $.
+    wfr2aOLD.3 $e |- F = wrecs ( R , A , G ) $.
+    $( Obsolete proof of ~ wfr2a as of 18-Nov-2024.
+       (New usage is discouraged.)  (Proof modification is discouraged.)
+       (Contributed by Scott Fenton, 30-Jul-2020.) $)
+    wfr2aOLD $p |- ( X e. dom F ->
+    ( F ` X ) = ( G ` ( F |` Pred ( R , A , X ) ) ) ) $=
+      ( vx cv cfv cpred cres wceq cdm fveq2 predeq3 reseq2d fveq2d eqeq12d
+      wfrlem12OLD vtoclga ) IJZCKZCABUCLZMZDKZNECKZCABELZMZDKZNIECOUCENZUDUHUGU
+      KUCECPULUFUJDULUEUICABUCEQRSTIABCDFGHUAUB $.
+  $}
+
+  ${
+    $d A z $.  $d F z $.  $d R z $.
+    wfr1OLD.1 $e |- R We A $.
+    wfr1OLD.2 $e |- R Se A $.
+    wfr1OLD.3 $e |- F = wrecs ( R , A , G ) $.
+    $( Obsolete proof of ~ wfr1 as of 18-Nov-2024.  (New usage is discouraged.)
+       (Proof modification is discouraged.)  (Contributed by Scott Fenton,
+       22-Apr-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    wfr1OLD $p |- F Fn A $=
+      ( vz wfn wfun cdm wceq wfrfunOLD cv cpred cres cfv cop csn cun eqid df-fn
+      wfrlem16OLD mpbir2an ) CAICJCKALABCDEFGMHACHNZCABUEOPDQRSTZBCDEFGUFUAUCCA
+      UBUD $.
+  $}
+
+  ${
+    $d A x $.  $d F x $.  $d G x $.  $d R x $.
+    wfr2OLD.1 $e |- R We A $.
+    wfr2OLD.2 $e |- R Se A $.
+    wfr2OLD.3 $e |- F = wrecs ( R , A , G ) $.
+    $( Obsolete proof of ~ wfr2 as of 18-Nov-2024.  (New usage is discouraged.)
+       (Proof modification is discouraged.)  (Contributed by Scott Fenton,
+       18-Apr-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    wfr2OLD $p |- ( X e. A ->
+                  ( F ` X ) = ( G ` ( F |` Pred ( R , A , X ) ) ) ) $=
+      ( vx wcel cdm cfv cpred cres wceq cv cop csn cun eqid wfrlem16OLD eleq2i
+      wfr2aOLD sylbir ) EAJECKZJECLCABEMNDLOUEAEIACIPZCABUFMNDLQRSZBCDFGHUGTUAU
+      BABCDEFGHUCUD $.
+  $}
+
+  ${
+    wfrrel.1 $e |- F = wrecs ( R , A , G ) $.
+    $( The well-ordered recursion generator generates a relationship.  Avoids
+       the axiom of replacement.  (Contributed by Scott Fenton, 8-Jun-2018.)
+       (Proof shortened by Scott Fenton, 17-Nov-2024.) $)
+    wfrrel $p |- Rel F $=
+      ( c2nd ccom cwrecs cfrecs df-wrecs eqtri frrrel ) ABCDFGZCABDHABMIEABDJKL
+      $.
+    $( $j usage 'wfrrel' avoids 'ax-rep'; $)
+
+    $( The domain of the well-ordered recursion generator is a subclass of
+       ` A ` .  Avoids the axiom of replacement.  (Contributed by Scott Fenton,
+       21-Apr-2011.)  (Proof shortened by Scott Fenton, 17-Nov-2024.) $)
+    wfrdmss $p |- dom F C_ A $=
+      ( c2nd ccom cwrecs cfrecs df-wrecs eqtri frrdmss ) ABCDFGZCABDHABMIEABDJK
+      L $.
+    $( $j usage 'wfrdmss' avoids 'ax-rep'; $)
+
+    $( Given ` F = wrecs ( R , A , X ) /\ X e. dom F ` , then its predecessor
```

Comment:
Smae as above: probably "(R, A, G)" but comment can probably lose a few symbols and be streamlined..

Reply:
Fixed